packages/tradershub/src/features/cfd/modals/MT5PasswordModal/ButtonGroups/AddAccountButtonsGroup.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Provider } from '@deriv/library';
+import { Button } from '@deriv/quill-design';
+import { ButtonGroup, Modal, SentEmailContent } from '../../../../../components';
+import { CFDPlatforms } from '../../../constants';
+import CreateAccountButton from './CreateAccountButton';
+
+type TProps = {
```

Comment:
rename
use one convention in some places you used TProps and other you use T[ComponentName]

Reply:
as I have noticed, other files are also using this naming convention

![image](https://github.com/binary-com/deriv-app/assets/108507236/d6b95756-3a75-4480-814c-3f07eb77b6f1)

In my understanding, if the prop type definition is scoped within the component itself, there is no need to complicate it by giving it long names i.e. `TDxtradePasswordModalProps` `TFooterComponentProps` `TPasswordComponentProps`